src/hdnode.js

```diff
@@ -51,7 +51,7 @@ HDNode.fromSeedHex = function (hex, network) {
   return HDNode.fromSeedBuffer(new Buffer(hex, 'hex'), network)
 }
 
-HDNode.fromBase58 = function (string, networks) {
+HDNode.fromBase58 = function (string, networks, skipValidation) {
```

Comment:
Yeah I understand. I could have added `function (string, options)`, and if `options` is a JS object take `options.network` and `options.skipValidation`; and if it is not, take it as network array.... but that could maybe break if `networks` is in fact just one network... and it sounds hacky.

Yeah I am not sure what to do with it.